src/main/java/dev/emi/trinkets/api/SlotType.java

```diff
@@ -156,4 +156,8 @@ public boolean equals(Object o) {
 	public int hashCode() {
 		return Objects.hash(group, name);
 	}
+
+	public String getId() {
```

Comment:
Why is this getting added to API?

Reply:
Since it is used in a lot of places (and now more due to vanilla enchantment changes) I though it made sense to do this. But I can remove that if you prefer.

Reply:
Nah you're right this is good to have. Weird to call it an ID when it's not an identifier, but I'll live.